### PR TITLE
Added search by Name

### PR DESCRIPTION
**Pull request by corneliouzbett**

### CODE REVIEW

File: api/src/main/java/org/openmrs/module/fhir2/api/dao/impl/FhirPersonDaoImpl.java [hunk 1]
```diff
@@ -10,12 +10,19 @@
...
 import lombok.Setter;
+import org.hibernate.SessionFactory;
+import org.hibernate.criterion.MatchMode;
 import org.openmrs.Person;
 import org.openmrs.api.PersonService;
 import org.openmrs.module.fhir2.api.dao.FhirPersonDao;
 import org.springframework.stereotype.Component;
 
 import javax.inject.Inject;
+import javax.inject.Named;
```
Comment (ibacher) [inline api/src/main/java/org/openmrs/module/fhir2/api/dao/impl/FhirPersonDaoImpl.java]: Is this import needed?

File: api/src/main/java/org/openmrs/module/fhir2/api/dao/impl/FhirPersonDaoImpl.java [hunk 2]
```diff
@@ -10,12 +10,19 @@
...
+import java.util.Collection;
+import java.util.List;
+
+import static org.hibernate.criterion.Restrictions.ilike;
```
Comment (ibacher) [inline api/src/main/java/org/openmrs/module/fhir2/api/dao/impl/FhirPersonDaoImpl.java]: Is this import needed?

File: api/src/main/java/org/openmrs/module/fhir2/api/dao/impl/FhirPersonDaoImpl.java [hunk 3]
```diff
@@ -10,12 +10,19 @@
 
 import lombok.AccessLevel;
 import lombok.Setter;
+import org.hibernate.SessionFactory;
```
Comment (ibacher) [inline api/src/main/java/org/openmrs/module/fhir2/api/dao/impl/FhirPersonDaoImpl.java]: Is this import needed?

File: api/src/main/java/org/openmrs/module/fhir2/api/dao/impl/FhirPersonDaoImpl.java [hunk 4]
```diff
@@ -10,12 +10,19 @@
...
+import org.hibernate.criterion.MatchMode;
```
Comment (ibacher) [inline api/src/main/java/org/openmrs/module/fhir2/api/dao/impl/FhirPersonDaoImpl.java]: Is this import needed?

**Comment by corneliouzbett** [comment]
@ibacher I have updated the PR

**Pull request merged**